vital/CMakeLists.txt

```diff
@@ -74,6 +74,7 @@ set( vital_public_headers
   attribute_set.h
   context.h
   exceptions.h
+  flags.h
```

Comment:
`flags` is such a generic name. Would you consider `bitmap_flags`?

Reply:
I'm trying to copy [QFlags](https://doc.qt.io/qt-5/qflags.html), FWIW. It's also just `flags` in ViViA.

What about `bitflags`? Or `bitmask`?

Reply:
`bitflags` works for me

Reply:
`bitflags` it is...